email-templates/booking-confirmed-en_US/booking-confirmed.text.ejs

```diff
@@ -0,0 +1,49 @@
+<% if (approvalrequired) { %>
+BOOKING REQUEST SENT
+You will receive a confirmation email once the organizer approves your request.
+<% } else { %>
+YOUR BOOKING IS CONFIRMED!
+<% } %>
+
+<%= event.name %>
+EVENT HOSTED BY <%= dojo.name %> [<%= process.env.URL_BASE %>dojos/<%= dojo.id %>]
+
+EVENT DETAILS
+<%= eventDate %> | <%= eventTime %>
+
+<%= event.address %>
+
+<% if (event.position) { %>
+Open in Google Maps [https://www.google.com/maps/search/?api=1&query=<%= event.position.lat %>,<%= event.position.lng %>]
+<% } else { %>
+Open in Google Maps [https://www.google.com/maps/search/?api=1&query=<%= encodeURIComponent(dojo.address) %>]
+<% } %>
+
+ATTENDEES
+<% for (let i = 0; i < applications.length; i += 1) { %>
+<%= applications[i].name %>
+<%= applications[i].tickettype %> / <%= applications[i].ticketname %>
```

Comment:
lowercase will fail :(

Reply:
Ugh. More reasons to not use html-to-text :(